vcpkg/ports/ktx/portfile.cmake

```diff
@@ -0,0 +1,85 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO KhronosGroup/KTX-Software
+    REF "v${VERSION}"
+    SHA512 5a89f8986464705ec36ac1becaddd0ff335e4c3c235468aaef0e963fcfeda4c0d669a086b91e61c16a3ae9e1fa5bf456dccf12cc65720e1a22e7cc0f30552541
+    HEAD_REF master
+    PATCHES
+        0001-Use-vcpkg-zstd.patch
+        0002-Fix-versioning.patch
+        0003-mkversion.patch
+        0004-quirks.patch
+        0005-no-vendored-libs.patch
+        CESIUM-0001-do-not-define-_DEBUG.patch
+        CESIUM-0002-not-a-framework.patch
+        CESIUM-0003-ios-normal-header-path.patch
+)
+file(REMOVE "${SOURCE_PATH}/other_include/zstd_errors.h")
+
+vcpkg_list(SET OPTIONS)
+# if(VCPKG_TARGET_IS_WINDOWS)
```

Comment:
Do we need to keep this?

Reply:
Probably not... I just copied it straight from Cesium for Unity

Reply:
Actually I think I'll keep the commented block, since it makes it clear how our portfile deviates from https://github.com/microsoft/vcpkg/blob/master/ports/ktx/portfile.cmake

Reply:
Sounds good since there's a reason to keep it.